sdk/formrecognizer/azure-ai-formrecognizer/src/test/resources/session-records/recognizeBadContentTypeArgument.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is this test removed?

Reply:
Yes. It is removed. I will delete this record json file